proposals/0079-allow_commission_decrease_at_any_time.md

```diff
@@ -0,0 +1,83 @@
+---
+simd: '0079'
+title: Allow Commission Decrease at Any Time
+authors:
+  - Bryan Ischo (bryan@ischo.com)
+category: Standard
+type: Core
+status: Draft
+created: 2023-10-26
+feature: TBD
+---
+
+## Summary
+
+The commission_updates_only_allowed_in_first_half_of_epoch feature disallows
```

Comment:
Please link the feature here https://github.com/solana-labs/solana/issues/29361

Reply:
Oh sorry, I think we had a misunderstanding -- I meant to mention the feature for `commission_updates_only_allowed_in_first_half_of_epoch` here. The feature that you mention in line 10 should be the *new* feature added for your change, not the previous one.

I created a feature gate issue for it which you can use in line 10: https://github.com/solana-labs/solana/issues/33994